test/e2e/storage/framework/testdriver.go

```diff
@@ -157,7 +157,7 @@ const (
 	CapSnapshotDataSource Capability = "snapshotDataSource" // support populate data from snapshot
 	CapPVCDataSource      Capability = "pvcDataSource"      // support populate data from pvc
 
-	// multiple pods on a node can use the same volume concurrently;
+	// CapMultiPODs on a node can use the same volume concurrently;
```

Comment:
This change is _not_ a fix to indentation or spacing.

Reply:
comment on exported type Repo should be of the form "XXX ..." (with optional leading article). So I changed it to match the spec！